test/powershell/Modules/Microsoft.PowerShell.Utility/Write-Debug.Tests.ps1

```diff
@@ -0,0 +1,17 @@
+Describe "Write-Debug tests" -Tags "CI" {
+    It "Should not have added line breaks" {
+        $text = "0123456789"
+        while ($text.Length -lt [Console]::WindowWidth) {
+            $text += $text
+        }
+        $origDebugPref = $DebugPreference
+        $DebugPreference = "Continue"
+        try {
+            $out = Write-Debug $text *>&1
+            $out | Should Match $text
```

Comment:
Maybe use "$out | Should BeExactly $longtext"?

Reply:
Same problem as above in that redirection currently adds a BOM